x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -777,7 +777,13 @@ public static RoleDescriptor kibanaSystemRoleDescriptor(String name) {
                         "metrics-apm-*",
                         "metrics-apm.*-*",
                         "traces-apm-*",
-                        "traces-apm.*-*"
+                        "traces-apm.*-*",
+                        "synthetics-http-*",
+                        "synthetics-icmp-*",
+                        "synthetics-tcp-*",
+                        "synthetics-browser-*",
+                        "synthetics-browser.network-*",
+                        "synthetics-browser.screenshot-*"
```

Comment:
__question:__ do we document these patterns anywhere like [we do for APM patterns](https://www.elastic.co/guide/en/apm/guide/8.1/apm-data-streams.html) (`dataset` part in particular, `http/icmp/tcp/browser/browser.network/browser.screenshot`)? I wanted to make sure there are no typos in the names and so on, but couldn't quickly find any references.

Reply:
Unfortunately no, we do have this docs ticket where we will address in the future https://github.com/elastic/synthetics/issues/286